features/update/basic/feature.ts

```diff
@@ -0,0 +1,53 @@
+import { WorkflowUpdateFailedError } from '@temporalio/client';
+import { Feature } from '@temporalio/harness';
+import * as wf from '@temporalio/workflow';
+import * as assert from 'assert';
+
+const WORKFLOW_INITIAL_STATE = '';
+const UPDATE_ARG = 'update-arg';
+const BAD_UPDATE_ARG = 'reject-me';
+const UPDATE_RESULT = 'update-result';
+const UPDATE_RESULT_NOT_YET_RECEIVED = 'update-result-not-yet-received';
+
+const myUpdate = wf.defineUpdate<Promise<string>, [string]>('myUpdate');
```

Comment:
Per https://github.com/temporalio/sdk-typescript/pull/1277, shouldn't this be:

```suggestion
const myUpdate = wf.defineUpdate<string, [string]>('myUpdate');
```

and `Promise` is implied? I wonder if we want to use TS typing tricks to prevent people from explicitly providing `Promse` as a ret type here.